fvtest/compilertriltest/MaxMinTest.cpp

```diff
@@ -85,7 +120,7 @@ TEST_P(Int32MaxMin, UsingLoadParam) {
 
 
 INSTANTIATE_TEST_CASE_P(MaxMin, Int32MaxMin, ::testing::Combine(
-    ::testing::ValuesIn(TRTest::combine(TRTest::const_values<int32_t>(), TRTest::const_values<int32_t>())),
+    ::testing::ValuesIn(TRTest::combine(intMaxMinTestValues(), intMaxMinTestValues())),
```

Comment:
Why not combine both of these? Or better yet why don't we scrap the two custom functions you have here (`intMaxMinTestValues` and `longMaxMinTestValues`) and put the constants directly in `TRTest::const_values` so all other tests get the benefit of testing the additional constants?